vignettes/dark_ink_transition/lib/demo.dart

```diff
@@ -17,8 +17,8 @@ class DarkInkDemo extends StatefulWidget {
 
 class _DarkInkDemoState extends State<DarkInkDemo> {
 
-  ValueNotifier<bool> _darkModeValue;
-  ScrollController _scrollController;
+  late ValueNotifier<bool> _darkModeValue;
+  late ScrollController _scrollController;
```

Comment:
Would prefer to just define these here rather than use `late`, ie
```dart
  ValueNotifier<bool> _darkModeValue = ValueNotifier<bool>(false);
  ScrollController _scrollController = SyncScrollController();
```